tests/testsuite/cargo_new/not_inherit_workspace_package_table_if_not_members/out/Cargo.toml

```diff
@@ -1,3 +1,5 @@
+[workspace]
+members = ["foo"]
 [workspace.package]
```

Comment:
Again, the question we need to answer: how much formatting should we do vs defer to `cargo fmt` like `cargo fix`

In this case, we normally do newlines between tables, but that'd require you to detect when adding `members` affects that.
- In this case, we'd need `members` to have a newline in its decor suffix
- In other cases (like if `resolver = "2"` is present), we'll need to remove the decor suffix on the last field and add it to `members`